api.php

```diff
@@ -22,4 +22,9 @@ function formatQuote($quote, $array)
 		
 		return $quote;
 	}
+	
+	function getHookUrl()

```

Comment:
Shouldn't this be in password.php instead? Maybe should rename it to passwords.php now as well since it will now be holding 3 different things.

Reply:
I removed the method from api.php, but moving and renaming is your job as password.php is ignored.